build_feathers.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+# Build feather packages
+# Run with:
+#    docker run --rm -v $PWD:/io quay.io/pypa/manylinux1_x86_64 /io/build_feathers.sh
+# or something like:
+#    docker run --rm -e PYTHON_VERSIONS=2.7 -v $PWD:/io quay.io/pypa/manylinux1_x86_64 /io/build_feathers.sh
+# or:
+#    docker run --rm -e FEATHER_VERSIONS=0.17.0 -e PYTHON_VERSIONS=2.7 -v $PWD:/io quay.io/pypa/manylinux1_x86_64 /io/build_feathers.sh
+#
+# Make sure numpy and Cython wheels are on the manylinux server or built in the
+# $WHEELHOUSE directory first.
+set -e
+
+# Manylinux, Python versions
+if [ -z "${PYTHON_VERSIONS}" ]; then
+    PYTHON_VERSIONS="2.7 3.4 3.5"
+fi
```

Comment:
Could do - I use this list for when I'm building lots of versions, for upload to the nipy server - or if projects will accept them - like numpy.  But you could do both - skip versions you know cannot build in the main loop.